eppo_client/client.py

```diff
@@ -334,6 +336,10 @@ def get_bandit_action_detail(
             )
             return BanditResult(variation, None)
 
+        # if no actions are given - a valid use case - return the default value
```

Comment:
We don't want to compute the flag string assignment (because it will log the flag assignment) and then return the default value if no actions are passed. If no actions are passed _and_ the flag is not associated with bandits, we can compute and return the string assignment.
This requires reading the [ `bandits` field of the UFC response](https://github.com/Eppo-exp/sdk-test-data/blob/282589ae9cd5f1382c0f9470618399bade934bc7/ufc/bandit-flags-v1.json#L155).

Reply:
This comment was correct at the time, but has since changed. See this [Slack thread](https://eppo-group.slack.com/archives/C05R6BK5BB9/p1721149341928549).

I've implemented the new plan, and have re-requesting a review